React-Native/en/1.3.3.md

```diff
@@ -0,0 +1,5 @@
+# [Success Criterion 1.3.3 - Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html)
+
+When designing an app it is important not to convey the usage through sensory characteristics. For example do not tell the user to click a button "to the right" or "the green link below". For users using a screenreader this will not convey any useful information.
+
+Considering this criterion will not only benefit your users who need assistive technologies, but also will allow your app to be localized for languages where layouts do not flow left to right (i.e. Chinese, Arabic).
```

Comment:
Use 'Meeting' instead of 'Considering'?

Meeting this criterion does not only benefit your users who need assistive technologies, but will also allow your app to be localized for languages where layouts do not flow left to right (i.e. Chinese, Arabic).